src/test/resources/smoke/axiom.vpr

```diff
@@ -0,0 +1,10 @@
+domain Foo {
+  axiom unsound {
+    false
+  }
+}
+
+method test()
+{
+  // refute.failed:refutation.true expected when smoke detection enabled
```

Comment:
I think if you set up the tests the way we discussed, you can use the actual ``//:: ExpectedOutput`` annotations here.

Reply:
-> d4f3e878